### DISCUSSION TITLE
Trying to understand how does Vorta works as a Borg frontend

### DISCUSSION THREAD

**Discussion by ReinerSanders** [borgbase/vorta#1907] [FAQ]
Hi there,

First of all, congratulations for such amazing software. I just started using Vorta and it's really great! Thank you very much! :)

So, once I will start backing up my things with Vorta / Borg, I would like to better understand one thing, please, if possible.

I assumed that Vorta is, like I read in some place, a frontend to Borg Backup. So I tried running some Borg commands, like `borg list` and `borg mount`, for example.

But, despite my repository is already created, and the first backup was already done, Borg still saying that there's no repository created, that the repository I specify does not exists, etc. Why this happens, please?

Well, once again, thank you very much, guys!

**Comment by goebbe** [comment]
@ReinerSanders There are different possible reasons, why it does not work as expected, in your case. Unfortunately your post does not contain enough information, so I do some guesswork. 
Which OS do you use? How did you install Vorta? (flatpak?) Is your repo on a server, or not? 

Things to check: 
- you should be able to run borg on your local computer, from the command line. To check this, type:  
  `borg --version` 
- if you do remote backups, make sure you can connect to the backup-server from you local computer using ssh:
   `ssh your_remote_username@yourbackupserver.com`
- use exactly the same "repo adress and path" as used in Vorta, don't forget "extra borg arguments", if you have specified any. 
  Working example on my local machine: 
  `borg list ssh://your_remote_username@yourbackupserver.com:22/~/path_to_repo/your_vorta_repo`
  then borg connects to your backup server and asks for the "passphrase" of your repo. Type your (case sensitive) passphrase and voila. I just tested this on my computer and it worked out of the box.

**Comment by ReinerSanders** [comment]
Hi @goebbe ,

Thank you very much. I'll take a look.
